classes/person.rb

```diff
@@ -1,8 +1,9 @@
-class Person
+class Person < Nameable
```

Comment:
- [ ] Kindly use `require_relative 'nameable'` to import the nameable class as this will cause an error when trying to create a new person.
![image](https://user-images.githubusercontent.com/24703171/233347625-50577619-533c-4cc7-a7e5-2913cbe3c07e.png)

Reply:
Okay, I am going to work on it right away. Thanks @gemmen29